implementation/jobberknoll/api/shared/mappers/account-mapper.ts

```diff
@@ -0,0 +1,12 @@
+import type { Account } from "@jobberknoll/core/domain";
+import type { AccountDto } from "~/shared/contracts/account-dto.ts";
+
+export function mapAccountToDto(account: Account): AccountDto {
+  return ({
+    id: account.id,
+    type: account.type,
+    fullName: account.fullName,
+    email: account.email,
+    phoneNumber: "phoneNumber" in account ? account.phoneNumber : undefined,
```

Comment:
Nope `.` works for possibly-`undefined` values, but not possibly-nonexistent values, and `phoneNumber` is not `undefined` in other `Account` types, it is missing at all.

![obraz](https://github.com/user-attachments/assets/efc24dc3-291b-4063-a0d5-44caa3615c52)

(to avoid the next question, `?.` wouldn't work either)